StorageUtility/Windows/StorageUtility.cpp

```diff
@@ -203,6 +203,30 @@ eErrorCode GetStorageAdapterProperty( HANDLE Handle, sStorageAdapterProperty& Ad
     return( eErrorCode::None );
 }
 
+eErrorCode GetPhysicalDiskNumber(HANDLE Handle, U32& PhysicalDiskNumber)
```

Comment:
DevicePath is unique but it's relied on some information of IdentifyDevice. In some cases, a drive may change the IdentifyDevice, so that the DevicePath will be also changed.
The PhysicalDiscNumber is hardware unique.